trj_loader_test: 
  trj_names: ["trj_test"]
  group_name: "manipulator"
  repetitions: 5
  recompute_time_law: false
  scaling: 0.1
  simulation: false
